src/components/structures/ThreadPanel.tsx

```diff
@@ -171,7 +171,9 @@ const ThreadPanel: React.FC<IProps> = ({
     const [narrow, setNarrow] = useState<boolean>(false);
 
     useEffect(() => {
-        setRoom(mxClient.getRoom(roomId));
+        const room = mxClient.getRoom(roomId);
+        setRoom(room);
+        room.fetchRoomThreads();
     }, [mxClient, roomId]);
```

Comment:
this may be saner as a `useMemo` given its entirely synchronous

Reply:
hm not sure `useMemo` would be needed here, it's a very cheap operation.  In this scenario `useEffect` and `useMemo` should be equal